docs/source/tracking.rst

```diff
@@ -91,6 +91,10 @@ directory. The URI defaults to ``mlruns``.
 :py:func:`mlflow.create_experiment` creates a new experiment and returns its ID. Runs can be
 launched under the experiment by passing the experiment ID to ``mlflow.start_run``.
 
+:py:func:`mlflow.set_experiment` sets an experiment as active. If the experiment does not exist,
+creates a new experiment. If you do not specify an experiment in :py:func:`mlflow.start_run`, new
```

Comment:
Maybe change to just "By default, new runs are launched under this experiment." More concise, but gets the message across -- it is unlikely that someone would want to use both `mlflow.set_experiment` and `mlflow.start_run` providing an experiment id simultaneously, and I think if you do write code that looks like that, the behavior works as expected (and we can document this in `mlflow.start_run`